src/advanced/service-testing.md

```diff
@@ -1,5 +1,9 @@
 # Service Testing
 
+!!! note
+    TestKit is also available in Java language.  
```

Comment:
I'd integrate it better (a note in the very beginning does not look good):

"You can test Exonum services with the help of the Exonum Testkit. [The rest of the paragraph
with the general description]

This document describes testing of Rust services with exonum-testkit crate, for Java 
instructions — see the \<section name> section."